app/views/idv/how_to_verify/show.html.erb

```diff
@@ -0,0 +1,79 @@
+<% title t('doc_auth.headings.how_to_verify') %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.how_to_verify')) %>
+<p><%= t('doc_auth.info.how_to_verify') %></p>
+
+<%= simple_form_for(
+      @idv_how_to_verify_form,
+      html: { autocomplete: 'off' },
+      method: :put,
+      url: idv_how_to_verify_url,
+    ) do |f|
+%>
+  <div role="group" aria-label="<%= t('doc_auth.headings.how_to_verify') %>">
+    <%= radio_button_tag(
```

Comment:
Can we use the form helpers instead of manually crafting HTML here, i.e. `f.input` ?